tests/input/example-class.php

```diff
@@ -5,8 +5,8 @@
 namespace Example;
 
 use Doctrine\Sniffs\Spacing\ControlStructureSniff;
```

Comment:
Maybe it's worth to add another class reference to verify the alphabetical sorting?